pillar/kube-minion.sls

```diff
@@ -1,2 +1,4 @@
 mine_functions:
   network.ip_addrs: [eth0]
+# infra container to use instead of downloading gcr.io/google_containers/pause
+pod_infra_container_image: sles12/pause:1.0.0
```

Comment:
A PR with just a variable declaration does not make sense. This variable must be used somewhere, otherwise for me it is a spurious declaration.